src/api/app/assets/stylesheets/webui/application/announcements.scss

```diff
@@ -0,0 +1,50 @@
+// Place all the styles related to the announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+#announcements {
+  .announcement {
+    background-color: #fff6ef;
+    border: 1px solid #d2702e;
+    padding: 1.7% 1%;
+    margin-bottom: 15px;
+    text-align: center;
+
+    h3 {
+      margin: 0;
+      color: #d2702e;
+    }
+
+    p {
+      margin-bottom: 10px;
+      font-size: 0.9em;
+      color: #777;
+
+      a {
+        font-size: 0.8em;
+        color: #069;
+        &:hover { color: #690; }
+      }
+    }
+  }
+
+
+  // Gives button appearance to the link
+  a.btn-default {
```

Comment:
`btn-default` is just a name I gave to the class, not related to bootstrap. I can change it if it's confusing.

I only found a few buttons on the views, all of them were inside forms (grey, with gradient, etc.) In ToS I had a link and I wanted it to look like a button. How? I had two options:

1) use a form instead of a link (and CSS for the button come from bento)
2) create a class for links to imitate buttons (`btn-default` in this case)

I think 2) is nicer :angel:

Reply:
Okay, nicer why?

Reply:
I mean, nicer than adding two forms in two different views for that (Announcement#show and Webui layout). The link needs only one line, the form needs at least two. We are not forced to use a form (no fields to fill in) just calling the controller method...

I would say it is more clean, maybe I'm wrong... Arguments for `form`?

Reply:
More lines but way easier to understand why you choose this. We are not in the line saving business :smile: